### PR TITLE
Some bless/autobless changes

### PR DESCRIPTION
**Pull request by oli-obk**
The autobless still won't work, as it doesn't have permissions, but at least it should do minimal work now

### CODE REVIEW

**Comment by nikomatsakis** [comment]
@oli-obk you have perms now to adjust things, so shall we assume you're just going to tweak this PR until it works?

File: test-crates/duchess-java-tests/tests/ui/duplicate_implements.stderr [hunk 1]
```diff
@@ -1,5 +1,5 @@
 error: error in class `log.Builder`: duplicate reference in 'implements' to 'log.BuildStep'
- --> $DIR/duplicate_implements.rs:4:5
```
Comment (rcoh) [inline test-crates/duchess-java-tests/tests/ui/duplicate_implements.stderr:2]: it's ok to replace `$DIR` with `test/ui` here?

**Review by rcoh — COMMENTED** [review]
Can we add a README for why we have these files? I would definitely otherwise assume that they are load bearing, but it seems like they're just information?

File: test-crates/duchess-java-tests/tests/ui/duplicate_implements.stderr (hunk 1 above)
Comment (oli-obk) [reply test-crates/duchess-java-tests/tests/ui/duplicate_implements.stderr:2]: yea, this is a leftover from the last ui_test version bump

**Review by rcoh — APPROVED** [review]
lgtm, we can also switch to using a rust-toolchain.toml, maybe the better long term solution

**Pull request merged**